storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -312,6 +312,13 @@
     // the topology.
     public static final String TOPOLOGY_SCHEDULER_STRATEGY = "topology.scheduler.strategy";
 
+    /**
+     * When DefaultResourceAwareStrategy or GenericResourceAwareStrategy is used,
+     * it will arrange unassigned executors based a particular order.
+     * If this config is set to true, the arrangement will be made by network proximity needs.
+     */
+    public static final String TOPOLOGY_RAS_ORDER_EXECUTORS_BY_PROXIMITY_NEEDS = "topology.ras.order.executors.by.proximity.needs";
```

Comment:
1. I was using`Boolean orderByProximity = (Boolean) td.getConf().getOrDefault(Config.TOPOLOGY_RAS_ORDER_EXECUTORS_BY_PROXIMITY_NEEDS, false);` in BaseResourceAwareStrategy. Both way can show this config in topology UI. Now I changed to using defaults.yaml.

2. Changed the name to include "schedule". I still keep "ras" since this is only used in RAS scheduling.

Reply:
Two places in the code set default values for the same config. Maybe we should delete this to avoid redundancy.